### PR TITLE
Update test_jinja2.py

### PR DESCRIPTION
**Pull request by rama280290**
Cross-site scripting (XSS) attacks can occur if untrusted input is not escaped. This applies to templates as well as code. The jinja2 templates may be vulnerable to XSS if the environment has autoescape set to False. Unfortunately, jinja2 sets autoescape to False by default. Explicitly setting autoescape to True when creating an Environment object will prevent this.

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [ ] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [ ] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [ ] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Comment by rama280290** [comment]
@ocelotl Kindly review and merge this please

**Review by ocelotl — APPROVED** [review]

**Comment by ocelotl** [comment]
Please allow maintainers to edit your PR or it will be very hard to merge.

**Comment by rama280290** [comment]
How to allow maintainers to edit this PR?

**Review by xrmx — APPROVED** [review]

**Review by emdneto — APPROVED** [review]

**Comment by xrmx** [comment]
@rama280290 please update the branch

**Comment by rama280290** [comment]
Updated the branch

**Comment by xrmx** [comment]
@rama280290 If you could rebase it once more, I've enable auto merge to get this merged. Thanks!

**Pull request merged**
